egs/wsj/s5/steps/segmentation/convert_ali_to_vec.pl

```diff
@@ -0,0 +1,17 @@
+#! /usr/bin/perl
+
+# Converts a kaldi integer vector in text format to 
```

Comment:
Converts an archive of Kaldi integer vectors in text format
to an archive of Kaldi floatint-point vectors in text format...


Clarify that it writes to the stdout.